pkg/controllers/metrics/node/suite_test.go

```diff
@@ -99,14 +104,30 @@ var _ = Describe("Node Metrics", func() {
 			Expect(metric.GetGauge().GetValue()).To(BeNumerically("~", v.AsApproximateFloat64()))
 		}
 	})
-	It("should remove the node metric gauge when the node is deleted", func() {
-		resources := corev1.ResourceList{
-			corev1.ResourcePods:   resource.MustParse("100"),
-			corev1.ResourceCPU:    resource.MustParse("5000"),
-			corev1.ResourceMemory: resource.MustParse("32Gi"),
-		}
+	It("should update the node lifetime and cluster utilization metrics", func() {
+
+		nodeCreationTimestamp := v1.NewTime(time.Now())
```

Comment:
I don't think you get to control this -- this should be auto-set by the apiserver when the resource is created

Reply:
I wasn't sure if the testing Suite sets this or not.